odoo/tools/misc.py

```diff
@@ -744,7 +744,7 @@ def remove_accents(input_str):
     meaning of input_str and work only for some cases"""
     if not input_str:
         return input_str
-    input_str = ustr(input_str)
+    input_str = ustr(input_str.translate(str.maketrans({'đ': 'd', 'Đ': 'd'})))
```

Comment:
Should explain why we need this

Reply:
For Vendor Bill journals with Vietnamese names, such as `Hóa đơn Nhà cung cấp`. After replacing accented latin letters by an ASCII equivalent, the result is `hoa-đon-nha-cung-cap` which contain accented character (`đ`). An error occurred: `You cannot use anything else than unaccented latin characters in the alias address.`

Reply:
as @pedrobaeza suggested we should use a generic slugification instead to cover wider range of language.

Reply:
the thing is unicodedata.normalize *is* our generic slugification. I see no other way or to `.encode('ascii', 'replace').decode()` the final str to force ascii